src/Interface/Modules/Render/ES/Core.cc

```diff
@@ -47,6 +47,8 @@ namespace Render {
   ren::registerAll(*this);
   rendererRegisterAll(*this);
 
+  mCurrentTime = 0.0;
```

Comment:
```suggestion
  
```

Reply:
Initialize the four uninitialized members in the parameter list above, like so:
`ESCore::ESCore() : mCoreSequence(0), mCurrentTime(0), mFPS(0), mLastRealTime(0), r_(0.0f), g_(0.0f), b_(0.0f), a_(0.0f)`